pg_search/Cargo.toml

```diff
@@ -40,8 +40,8 @@ serde = "1.0.188"
 serde_json = "1.0.105"
 serde_path_to_error = "0.1.14"
 shared = { path = "../shared" }
-tantivy = { git = "https://github.com/paradedb/tantivy.git", package = "tantivy", rev = "e678820" }
-tantivy-common = { git = "https://github.com/paradedb/tantivy.git", rev = "e678820" }
+tantivy = { git = "https://github.com/paradedb/tantivy.git", package = "tantivy", rev = "c7f13ef" }
```

Comment:
@sardination Could we highlight in the top of the README the Tantivy version we're on? It's not obviously apparent based on the commit sha, and I know many users are interested. A lot of the people who follow us also follow Tantivy directly